Delcam.ProductInterface.PowerMILL.Test/PMEntityTests/PMProjectTests.cs

```diff
@@ -147,6 +147,24 @@ public void CreateBlockFromBoundaryTest()
             Assert.That(boundingBox.MinZ, Is.EqualTo((Autodesk.Geometry.MM)(-30.004846)));
         }
 
+        [Test]
+        public void ExportBlockTest()
+        {            
+            _powerMILL.ActiveProject.Boundaries.CreateBoundary(TestFiles.CurvesFiles);
+            _powerMILL.ActiveProject.Refresh();
+            PMBoundary boundary = _powerMILL.ActiveProject.Boundaries.ActiveItem;
+            var boundingBox = _powerMILL.ActiveProject.CreateBlockFromBoundaryWithLimits(boundary, 0, 100);
+            FileSystem.File file = new FileSystem.File(TestFiles.DIRECTORY + "\\ExportBlock.dmt");
```

Comment:
I would suggest using a file in the temporary directory instead.  The File class has a method on it to give you a name of a temporary file.

Reply:
learned something new every day.